wasm.md

```diff
@@ -117,7 +117,7 @@ When a binary operator is the next instruction, the two arguments are loaded fro
  //                | "(" FValType "." FBinOp ")" | FValType "." FBinOp Float Float
  // ------------------------------------------------------------------------------
     rule <k> ( ITYPE . BOP:IBinOp ) => ITYPE . BOP SI1 SI2 ... </k>
-         <stack> < ITYPE > SI1 : < ITYPE > SI2 : STACK => STACK </stack>
+         <stack> < ITYPE > SI2 : < ITYPE > SI1 : STACK => STACK </stack>
```

Comment:
Yeah, agree, didn't consider what `SI` stood for. Still, I feel it's more intuitive to think of them as "operand 1" and "operand 2", so that their name reflects their semantic meaning rather than what order they happen to be on the stack. Would it make sense to call them `C2` and `C1` instead (as in the spec)?

Reply:
Switched to C1 and C2 for now.

Reply:
Lets do the same for `ConvOp` then as well.

Reply:
Good call, missed that one